openedx/core/djangoapps/course_groups/tests/helpers.py

```diff
@@ -81,39 +78,19 @@ def config_course_cohorts_legacy(
 
     Arguments:
         course: CourseDescriptor
-        discussions: list of topic names strings.  Picks ids and sort_keys
-            automatically.
         cohorted: bool.
-        cohorted_discussions: optional list of topic names.  If specified,
-            converts them to use the same ids as topic names.
         auto_cohort_groups: optional list of strings
                   (names of groups to put students into).
 
     Returns:
         Nothing -- modifies course in place.
     """
-    def to_id(name):
-        """
-        Helper method to convert a discussion topic name to a database identifier
-        """
-        return topic_name_to_id(course, name)
```

Comment:
This is actually what prompted me to remove these unused args-- in my last PR, I moved topic_name_to_id, but didn't include the new import in this file. It caused no test errors because "to_id" ended up never being used, but I noticed in Pycharm that topic_name_to_id did not have a proper reference. Rather than add the new import, I deleted the unused code.